tests/stdlib/tstring.nim

```diff
@@ -0,0 +1,52 @@
+discard """
+  file: "tstring.nim"
+  output: '''
```

Comment:
I will accept this, but in general I don't like ``unittest`` for testament. Unnecessary and slows down compilations. There is nothing here that wouldn't work with ``doAssert`` instead.